_guides/deploying-with-ksonnet.md

```diff
@@ -0,0 +1,519 @@
+---
+title: Deploying with KSonnet
+collection: How-to Guides
+layout: article
+section: On-prem
+category: component
+order: 1
+since: 20190119
+---
+
+One of the first problems this article solves is simplifying how users configure the applications they deploy to Kubernetes clusters.
+
+With that in mind, we are representing a detailed step by step instruction on how to implement the {{site.data.tenant.name}} platform's cluster and all the required Kubernetes descriptors onto a customer's hardware utilizing KSonnet.
+
+Before we go ahead and dive into the configuration peculiarities, let's deal with some of the basic terminologies.
+
+## What is Kubernetes 
+
+> Kubernetes is an open source container orchestration tool developed by Google. [Source code in GitHub](https://github.com/kubernetes/kubernetes).
+
+Kubernetes is an open source toolkit for building a fault-tolerant, scalable platform designed to automate and centrally manage [containerized applications](https://www.techradar.com/news/what-is-container-technology).
+
+The platform itself can be deployed within almost any infrastructure – into any working Kubernetes cluster (manually installed into private infrastructure) in the local network, server cluster, data center, any kind of cloud – public (Google Kubernetes Engine, Azure Kubernetes Service, Amazon Elastic Container Service for Kubernetes, etc.), private, hybrid, or even over the combination of these methods.
+
+### Kubernetes Architecture
+
+To build a platform based on Kubernetes, we will need physical or virtual servers that perform the following roles:
+
+* *Node* is a separate host in the Kubernetes cluster, which is used to start worker threads and containerized applications;
+
```

Comment:
Please avoid this empty lines between bullet points. They make the list look different than in all other articles.

Reply:
Got it.